src/Fantomas.Core.Tests/MultilineBlockBracketsOnSameColumnRecordTests.fs

```diff
@@ -1395,3 +1395,61 @@ let f
     =
     x
 """
+
+[<Test>]
+let ``comment in bracket ranges of anonymous type, 2566`` () =
+    formatSourceString
+        false
+        """
+let x = {| // test1
+    Y = 42
+    Z = "string"
+    Foo = "Bar"
+    // test2
+    |}
+
+let y = {|
+    Y = 42
+    // test
+|}
+
+let z = {|
+    Y = 42
+|}
+
+let a = {| // test1
+    foo with
+    Level = 7
+    Square = 9
+    // test2
+|}
+"""
+        config
+    |> prepend newline
+    |> should
+        equal
+        """
+let x =
+    {| // test1
+        Y = 42
+        Z = "string"
+        Foo = "Bar"
+    // test2
+    |}
+
+let y =
+    {|
+        Y = 42
+    // test
+    |}
+
+let z = {| Y = 42 |}
+
+let a =
+    {| // test1
+    foo with
```

Comment:
I know it is the same as we have it for regular records.
But in the future, we might want to have something like
```fsharp
{| // foo
   bar with
    A = 1
    B = 2 |}
```

I don't feel strongly about this, so no need to address this in this PR.

Reply:
I think this should better be handled in a separate PR.

Reply:
Sounds good to me!
I have no idea why the CI is failing, the NuGet lock file should not be throwing errors.
I think we need to apply the steps explained [here](https://github.com/NuGet/Home/issues/7921#issuecomment-478152479).
I'll do that in a separate PR so you can rebase with main.

Reply:
There is also https://github.com/dotnet/fsharp/issues/13678.
I hope it is ok.